src/utils/iframe.js

```diff
@@ -4,6 +4,15 @@
  * @author Box
  */
 
+/**
+ * Check if the current window is inside an iframe
+ *
+ * @return {boolean}
+ */
+export function isInsideIframe() {
+    return window.top !== window.self;
```

Comment:
does this solution work for all browsers we support?

Reply:
seems like it should work

for reference I found an Azure implementation:
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/src/utils/BrowserUtils.ts#L43-L45